artview/components/tools.py

```diff
@@ -122,16 +123,18 @@ def onPick(self, event):
             if az < 0:
                 az = az + 360.
             rng = np.sqrt(xdata*xdata + ydata*ydata)
-            # TJL - Attempt to pep8 this overlong string
-            msg1 = 'x = %4.2f, y = %4.2f, ' % (xdata, ydata)
-            msg2 = 'Azimuth = %4.2f deg., Range = %4.3f km' % (az, rng)
-            self.msg = msg1 + msg2
+            self.msg = 'x = %4.2f, y = %4.2f, Azimuth = %4.2f deg., Range = %4.3f km'\
+                        %(xdata, ydata, az, rng)
         else:
-            az = np.arctan2(xdata, ydata) * 180. / np.pi
-            radar = self.Vradar.value  # keep equations clean
+            az = np.arctan2(xdata, ydata)*180./np.pi
+            radar = self.Vradar.value #keep equantions clean
             if az < 0:
                 az = az + 360.
             rng = np.sqrt(xdata*xdata + ydata*ydata)
+            # TJL - Attempt to pep8 this overlong string
+            msg1 = 'x = %4.2f, y = %4.2f, ' % (xdata, ydata)
+            msg2 = 'Azimuth = %4.2f deg., Range = %4.3f km' % (az, rng)
+            self.msg = msg1 + msg2
```

Comment:
Why is this different that above?

Also, can we keep spaces on mathematical operators?